routes/frontend.php

```diff
@@ -1,7 +1,12 @@
 <?php
 
 use Illuminate\Support\Facades\Route;
-use Workup\MenuBuilder\Http\Controllers\FrontendController;
+use Workup\MenuBuilder\Http\Controllers\Frontend\MenuController;
+use Workup\MenuBuilder\Http\Controllers\Frontend\MenuItemController;
 
-Route::get('/menus', [FrontendController::class, 'index']);
-Route::get('/menus/{menu}', [FrontendController::class, 'show']);
+Route::get('/menus', [MenuController::class, 'index']);
```

Comment:
I saw that this Controller is into swagger... Is this route visible to client frontend as API? If yes we should add:
\App\Http\Middleware\SiteConfig::class,
into frontend route kernel setting or manually add siteConfig middleware to each route!